### PR TITLE
Echoes: Logic Updates and Video Upload

### PR DESCRIPTION
**Pull request by CMos75**
#### logic database

- Added: 15 videos to the logic database. see the [Video Directory] (https://randovania.github.io/Metroid%20Prime%202%20Echoes/) for the full collection
- Added: Comments to some Beginner Bomb Jump tricks

##### Sanctuary Fortress:

- Added: Watch Staton: Bomb Space Jump (Advanced) to reach the Spidertrack (Standable Terrain (Advanced)) to reach the Bomb Slot with an Instant Morph (Advanced)

### CODE REVIEW

**Comment by Miepee** [comment]
Missing Changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
 - Changed: A proper error message is displayed when mono is not found, when exporting a game on macOS and Linux.
 
+#### Logic Database
+
+- Added: 15 videos to the logic database. see the [Video Directory]
```
Comment (dyceron) [inline CHANGELOG.md:69]: see -> See

File: CHANGELOG.md [hunk 2]
```diff
@@ -64,6 +64,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+(https://randovania.github.io/Metroid%20Prime%202%20Echoes/) for the full collection 
+- Added: Comments to some Beginner Bomb Jump tricks
+
+##### Sanctuary Fortress:
+
+- Added: Watch Staton: Bomb Space Jump (Advanced) to reach the Spidertrack (Standable Terrain (Advanced)) to reach the Bomb Slot with an Instant Morph (Advanced)
```
Comment (dyceron) [inline CHANGELOG.md:75]: Staton -> Station
Spidertrack -> Spider Track

```suggestion
- Added: Accessing the portal in Watch Station with a Bomb Space Jump (Advanced) to reach the Spider Track, Standable Terrain (Advanced) to reach the Bomb Slot, and an Instant Morph (Advanced)
```

File: randovania/games/prime2/json_data/Sanctuary Fortress.txt [hunk 3]
```diff
@@ -1281,19 +1293,22 @@ Extra - in_dark_aether: False
...
   * Layers: default
   > Door to Watch Station Access
+      # NSJ https://www.youtube.com/watch?v=z2zCtuLRRvc
       Space Jump Boots or Movement (Beginner)
   > Door to Central Area Transport West
       Trivial
   > Portal to Aerial Training Site
       After Watch Station Portal Gate
   > Door to Grand Abyss
+      # NSJ https://www.youtube.com/watch?v=z2zCtuLRRvc&t=8s
```
Comment (dyceron) [inline randovania/games/prime2/json_data/Sanctuary Fortress.txt:1311]: Possibly change these to say `No Space Jump` or `Without Space Jump`

**Review by dyceron — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dyceron) [reply CHANGELOG.md]: This still need adjusting

Comment (CMos75) [reply CHANGELOG.md]: I knew I'm bad at phrasing things lul. Should be good now tho.

**Review by dyceron — APPROVED** [review]

**Pull request merged**